tests/test_api.py

```diff
@@ -87,8 +112,120 @@ def setUp(self):
             print(resp)
```

Comment:
`print` might not be required in this line

Reply:
True, removed (+1)